src/components/molecules/PlanRoom/Battons/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, ButtonGroup, Box } from '@chakra-ui/react';
```

Comment:
You have a typo in folder name (`Battons` instead of `Buttons`).

Reply:
As for the buttons, I will leave them in a separate file already. I took the colour from our documentation. The name will be changed